src/modules/enrollment/hooks/useEnrollmentSummaryDialog.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+  Button,
+  DialogActions,
+  DialogContent,
+  DialogProps,
+  DialogTitle,
+} from '@mui/material';
+import { useCallback, useMemo, useState } from 'react';
+
+import CommonDialog from '@/components/elements/CommonDialog';
+import GenericTable from '@/components/elements/table/GenericTable';
+import { ColumnDef } from '@/components/elements/table/types';
+import HmisEnum from '@/modules/hmis/components/HmisEnum';
+import { entryExitRange, parseAndFormatDate } from '@/modules/hmis/hmisUtil';
+import { Routes } from '@/routes/routes';
+import { HmisEnums } from '@/types/gqlEnums';
+import { EnrollmentSummaryFieldsFragment, ProjectType } from '@/types/gqlTypes';
+import generateSafePath from '@/utils/generateSafePath';
+
+const baseColumns: ColumnDef<EnrollmentSummaryFieldsFragment>[] = [
+  {
+    header: 'Enrollment Period',
+    render: (e) => entryExitRange(e),
+  },
+  {
+    header: 'Project',
+    linkTreatment: true,
+    render: (e) => e.projectName,
+  },
+  {
+    header: 'Project Type',
+    render: (e) => (
+      <HmisEnum value={e.projectType} enumMap={HmisEnums.ProjectType} />
+    ),
+  },
+  {
+    header: 'Move in Date',
+    render: (e) => {
+      const projectTypes = [
+        ProjectType.Ph,
+        ProjectType.Oph,
+        ProjectType.Th,
+        ProjectType.Psh,
+      ];
+
+      if (projectTypes.includes(e.projectType))
+        return parseAndFormatDate(e.moveInDate);
+
+      return null;
+    },
+  },
+];
+
+const useEnrollmentSummaryDialog = ({
+  enrollmentSummary,
+  clientId,
+}: {
+  enrollmentSummary?: EnrollmentSummaryFieldsFragment[];
+  clientId?: string;
+}) => {
+  const [dialogOpen, setDialogOpen] = useState(false);
+
+  const closeDialog = useCallback(() => setDialogOpen(() => false), []);
+  const openDialog = useCallback(() => setDialogOpen(() => true), []);
+
+  const renderDialog = useCallback(
+    (props: Partial<DialogProps> = {}) => {
+      if (!enrollmentSummary) return null;
+
+      return (
+        <CommonDialog {...props} open={!!dialogOpen} onClose={closeDialog}>
```

Comment:
can we give this a fullWidth and maxWidth=md? Looks a little cramped.
<img width="1133" alt="Screen Shot 2023-07-27 at 4 27 57 PM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/652f13ba-b931-4bf8-9b32-8e9eac8098cb">